examples/advanced/node/config.js

```diff
@@ -0,0 +1,121 @@
+const StyleDictionary = require('style-dictionary');
+// Rather than have Style Dictionary handle the merging of property files,
+// you could use node module export/require to do it yourself. This will
+// allow you to not have to copy object namespaces like you normally would.
+// Take a look at any of the .js files in components/ or properties/
+const properties = require('./properties');
+
+const buildPath = 'build/';
+
+// You can still add custom transforms and formats like you
+// normally would and reference them in the config below.
+StyleDictionary.registerTransform({
+  name: 'test',
+  type: 'name',
+  transformer: function(prop) {
+    return prop.path.splice(2).join(' ')
+  }
+});
+
+StyleDictionary.registerFormat({
+  name: "testing",
```

Comment:
generally, in the examples, I tend to use more meaningful names (eg. `custom/android` or `custom/template/scss`), so the code is easier to read and follow. Here for example you could use `myRegisteredFormat`